src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,67 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+/*
+ * Tagging object class, to record list of commands for tags used in session
+ */
+
+public class Tagging {
+
+	public static final String _ADD = "+"; 
+	public static final String _REMOVE = "-";
+	
+	public static ArrayList<String> tagHistory;
+	
+	private String command;
+	private Person person;
+	private Tag tag;
+	
+	public Tagging ( String command, Person person, Tag tag ){
+		
+		if ( command.equalsIgnoreCase("add"))
+			command = _ADD;
+		else
+			command = _REMOVE;
```

Comment:
Coding standard violation! One more more of these problems detected: 
1. extra/missing spaces. 
2. brace style problem. 
3. missing braces.